src/vm/errors/vm_errors.rs

```diff
@@ -82,6 +86,9 @@ impl fmt::Display for VirtualMachineError {
                 write!(f, "Inconsistent auto-deduction for builtin {}, expected {:?}, got {:?}", builtin_name, expected_value, current_value)
             },
             VirtualMachineError::RunnerError(runner_error) => runner_error.fmt(f),
+            VirtualMachineError::HintException(address) => write!(f, "Got an exception while executing a hint at pc: {:?}", address),
```

Comment:
Is the PC enough for the user to figure out which hint raised? Otherwise, it may be a good idea to include the hint here.